### ISSUE TITLE
AU Base Practitioner Profile - old URL for v203 table

### ISSUE THREAD

**Issue by robeastwood-agency** [hl7au/au-fhir-base#300]
The identifier slice for HPI-I has a requiredPattern set for the type element as 
```
{"coding":[{"system":"http://hl7.org/fhir/v2/0203","code":"NPI"}]}
```
The v0203 codesystem has a new URL in R4: [http://terminology.hl7.org/CodeSystem/v2-0203](http://hl7.org/fhir/v2/0203/index.html). 

Note that changing this value eliminates 2 QA errors re the non-R4 URL

This is also the case for:
- au-patient, au-practitionerrole and au-relatedperson.xml

**Comment by oridashi** [comment]
see also #296

**Comment by robeastwood-agency** [comment]
This has been fixed a while back, via #**b25b1ec9216e47f1da1d45fd62c4ae783be34489**
